apps/cli/src/main.ts

```diff
@@ -1,20 +1,25 @@
 #!/usr/bin/env node
 
-import { Command, Option, OptionValues } from '@commander-js/extra-typings'
+import type { OptionValues } from '@commander-js/extra-typings'
+import { Command, Option } from '@commander-js/extra-typings'
+import type { TemplateType } from '@featureboard/code-generator'
 import {
     FsTree,
-    TemplateType,
     codeGenerator,
     flushChanges,
     printChanges,
 } from '@featureboard/code-generator'
-import * as figlet from 'figlet'
-import fs from 'fs/promises'
+import figlet from 'figlet'
+import fs from 'fs'
+import fsAsync from 'fs/promises'
 import path from 'path'
 import { exit } from 'process'
 import prompts from 'prompts'
-import * as packageJson from '../package.json'
 
+// dynamically load package.json using fs
+const packageJson: { version: string } = JSON.parse(
```

Comment:
Can you check that this still works as file placement changes between the built and unbuilt code
The import just hardcoded the version into the bundle

Reply:
The problem was that i changed the root to `src`, and package.json is outside the root of the project so TypeScript fails to compile. Will double check